packages/react-components/source/react/library/tag/Tag.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { node, func } from 'prop-types';
+import Button from '../button';
+import Text from '../text';
+
+const propTypes = {
+  /** Tag text or other content */
+  children: node.isRequired,
+  /** Callback function called when clode icon is clicked */
+  onClick: func,
+};
+
+const defaultProps = {
+  onClick: () => {},
+};
+
+const Tag = ({ children, onClick }) => {
+  return (
+    <div className="rc-tag">
+      <div className="rc-tag-label-background">
+        <Text className="rc-tag-text">{children}</Text>
+      </div>
+      <Button
+        className="rc-tag-remove-button"
+        onClick={() => onClick()}
+        icon="close"
+      />
```

Comment:
If we wanted to be even more accessible here, we could add an `aria-label` to the button (which makes me realize we should do that in the other icon button examples).

```suggestion
      <Button
        className="rc-tag-remove-button"
        onClick={() => onClick()}
        icon="close"
        aria-label="Remove tag"
      />
```

There's probably some way to use `aria-labelledby` to create a relationship between the Tag text and the button, but that might require `id` generation. I think the above use of `aria-label` would suffice.